horace_core/sqw/@sqw/symmetrise_sqw.m

```diff
@@ -264,7 +264,7 @@
 
         elseif all(cellfun(@(x) isa(x, 'SymopReflection') || ...
                                 isa(x, 'SymopIdentity'), sym))
-            sym = cell2mat(sym);
+            sym = cat(1, sym{:});
```

Comment:
noting that this forces a column vector, which is probably what you want, but cell2mat might give you a row vector under the same conditions. 

Once you've checked this, this is really an approval, go ahead and merge